gnome/evolution-data-server/Portfile

```diff
@@ -52,7 +52,7 @@ depends_lib         path:lib/pkgconfig/gtk+-3.0.pc:gtk3 \
                     port:db53
 
 # our choice of perl for installed scripts
-depends_lib-append  port:perl5.28
+depends_lib-append  port:perl5.34
```

Comment:
More work is needed, see https://trac.macports.org/ticket/65409

Reply:
> More work is needed, see https://trac.macports.org/ticket/65409

Fixed via separate commit: https://github.com/macports/macports-ports/commit/55d6436349def9ce35944a3e389f149329390f05

So changes to this port can be removed from the PR.